backend/routes/api/quickstarts/quickStartUtils.js

```diff
@@ -24,13 +25,17 @@ const getInstalledQuickStarts = async (fastify) => {
   //   fastify.log.error(e, 'failed to get quickstarts');
   // }
 
+  const featureFlags = features.getComponentFeatureFlags();
+
   // TODO: Remove local quick starts when we get the correct quick starts from OpenShift
   const normalizedPath = path.join(__dirname, '../../../../data/quickstarts');
   fs.readdirSync(normalizedPath).forEach((file) => {
     if (constants.yamlRegExp.test(file)) {
       try {
         const doc = jsYaml.load(fs.readFileSync(path.join(normalizedPath, file), 'utf8'));
-        installedQuickStarts.push(doc);
+        if (!doc.spec.featureFlag || featureFlags[doc.spec.featureFlag]) {
```

Comment:
future: this may also work with just the one check and optional chaining, I'd need to fully confirm. Regardless, what you have works, pre-opt chaining

Reply:
The idea is that if there is no feature flag it is allowed, if there is a feature flag, check the flags.